src/main/java/uk/gov/laa/ccms/caab/constants/SessionConstants.java

```diff
@@ -74,6 +74,8 @@ public class SessionConstants {
    */
   public static final String NOTIFICATION_SEARCH_CRITERIA = "notificationSearchCriteria";
 
+  public static final String NOTIFICATIONS = "notifications";
```

Comment:
Are these the results from the notification search, could it be renamed to notificationSearchResults

also please add javadoc description.

Reply:
will change the variable and add javadoc

Reply:
done